packages/test-utils/src/wrapper-array.js

```diff
@@ -38,21 +31,17 @@ export default class WrapperArray implements BaseWrapper {
   contains (selector: Selector): boolean {
     this.throwErrorIfWrappersIsEmpty('contains')
 
-    return this.wrappers.every(wrapper => wrapper.contains(selector))
+    return this.every(wrapper => wrapper.contains(selector))
   }
 
   exists (): boolean {
-    return this.length > 0 && this.wrappers.every(wrapper => wrapper.exists())
-  }
-
-  filter (predicate: Function): WrapperArray {
-    return new WrapperArray(this.wrappers.filter(predicate))
+    return this.length > 0 && this.every(wrapper => wrapper.exists())
   }
 
   visible (): boolean {
     this.throwErrorIfWrappersIsEmpty('visible')
```

Comment:
Personally I think we can remove `throwErrorIfWrappersIsEmpty`, and just return false when `length === 0`. Just like: `return this.length > 0 && this.every(wrapper => wrapper.visible())`